include/dxguids/dxguids.h

```diff
@@ -7,6 +7,8 @@
 #error "This header requires C++"
 #endif
 
+#include <d3d12video.h>
```

Comment:
This shouldn't be here. The pattern for this file is to include the relevant headers (e.g. d3d12.h, d3d12sdklayers.h, d3d12video.h) and then include dxguids.h afterwards.